h2o-genmodel/src/main/java/hex/genmodel/tools/PrintMojo.java

```diff
@@ -68,6 +71,10 @@ private static void usage() {
     System.out.println("");
     System.out.println("    --fontsize | -f    Set font sizes of strings.");
     System.out.println("");
+    System.out.println("    --raw    Print raw graph representation to standard output.");
```

Comment:
What is "raw"?

Reply:
Raw is a mode that has been there for a long time, added by @tomkraljevic . I've only added it to the documentation. It basically prints the graph in a more human-readable format. Examle:

```
Graph

    ----- Tree 0 -----
    Nodes
        Node 0
            weight:      150.0
            depth:       0
            colId:       0
            colName:     C5
            leftward:    true
            naVsRest:    false
            splitVal:    1.5
            isBitset:    true
            predValue:   0.3649551
            squaredErr:  11.1307125
            leftChild:   Node 1
            rightChild:  Node 4
        Node 4
            weight:      50.0
            depth:       1
            colId:       0
            colName:     
            leftward:    false
            naVsRest:    false
            splitVal:    NaN
            isBitset:    false
            predValue:   0.0
            squaredErr:  4.1470647
            leftChild:   
            rightChild:  
        Node 1
            weight:      100.0
            depth:       1
            colId:       0
            colName:     C5
            leftward:    false
            naVsRest:    false
            splitVal:    NaN
            isBitset:    true
            predValue:   0.54743266
            squaredErr:  6.983648
            leftChild:   Node 5
            rightChild:  Node 6
        Node 6
            weight:      50.0
            depth:       2
            colId:       0
            colName:     
            leftward:    false
            naVsRest:    false
            splitVal:    NaN
            isBitset:    false
            predValue:   1.0
            squaredErr:  3.7480965
            leftChild:   
            rightChild:  
        Node 5
            weight:      50.0
            depth:       2
            colId:       0
            colName:     
            leftward:    false
            naVsRest:    false
            splitVal:    NaN
            isBitset:    false
            predValue:   1.0
            squaredErr:  3.2347205
            leftChild:   
            rightChild:  

    Edges
        Node 0 ---left---> Node 1
        Node 1 ---left---> Node 5
        Node 1 ---right--> Node 6
        Node 0 ---right--> Node 4
```

Under this output, there is the graphviz format as well. I'll document it better.

Reply:
Yeah, I know that :) But the users won't know that from the description. I would either not expose the output (that would be my preference) or document what it does.

Reply:
Removed.